### ISSUE TITLE
A question about the paper

### ISSUE THREAD

**Issue by darkpromise98** [BIT-DA/MetaSAug#3]
In the equation (5), we compute the gradient of covariance matrix by the standard cross-entropy loss/equation (3) on validation set. But it seems that the standard cross-entropy function don't need to use the covariance matrix to compute the loss, so the related gradient should be zero ?
